src/services/request.service.js

```diff
@@ -73,6 +74,13 @@ const RequestService = {
       where: { UserId: userId },
       include: { model: db.Trips, attributes: { exclude: ['RequestId'] } } });
     return result;
+  },
+  async getTraveledDestinations() {
+    // eslint-disable-next-line no-return-await
+    return await db.Trips.findAll({ attributes: ['country', 'city', [Sequelize.fn('COUNT', 'city'), 'N of visitors']],
+      group: ['city', 'country'],
+      order: [[Sequelize.fn('COUNT', 'city'), 'DESC']]
+    });
   }
```

Comment:
I think we should also `exclude` the trips that belong to the requests that are `not yet approved`. or to be more specific, we can consider `done trips` only.

Reply:
we considered done trips only